### ISSUE TITLE
Duplicate names in normalize_entity's additional/copy_variables throws non-obvious error

### ISSUE THREAD

**Issue by georgewambold** [Featuretools/featuretools#347]
I'm a new user of featuretools and so far it's been awesome. The documentation is way above par for open source and it seems has a lot of potential to help me with my ml projects. 

While trying to convert a pandas DataFrame into a featuretools EntitySet using `entity_from_dataframe` followed by `normalize_entity` I got a strange error:  `'DataFrame' object has no attribute 'dtype'`. 

I spent a while trying to track down the issue and it turned out that in the `additional_variables` parameter of `normalize_entity` I had passed the same column/variable name twice. I had a lot of additional variables so the duplicate column wasn't obvious to me right away. 

Here's a small snippet to reproduce the error:
```
from featuretools.entityset import EntitySet
import pandas as pd

demo_df = pd.DataFrame({
    'test_score_id': [100,101,102],
    'score': [15,16,17],
    'user_id': [1,2,1],
    'name':['Ali', 'Joe', 'Ali'],
})

es = EntitySet()

es.entity_from_dataframe(
    entity_id='test_scores',
    dataframe=demo_df,
    index='test_score_id'
)

es.normalize_entity(
    base_entity_id='test_scores',
    new_entity_id='users',
    index='user_id',
    additional_variables=['name','name']
)
#=> 'DataFrame' object has no attribute 'dtype'
```

It would be nice to get a more descriptive error when normalize_entity gets duplicate column names. It might save someone a headache in the future.

Thanks for maintaining featuretools and the help with this issue!

**Comment by kmax12** [comment]
@georgewambold thanks for trying out featuretools and sharing this. I agree we should detect this situation and raise an error. 

if you (or anyone else reading this) is interested, we're open to outside contributions.  

You would just need to add a check inside of `normalize_entity` and also add a test case in `featuretools/tests/entityset_tests/test_es.py`.

Let me know if you're interested, otherwise we'll put it on the todo list to take care of.

**Comment by georgewambold** [comment]
I'd be happy to write the PR! Thanks for responding so quickly

**Comment by georgewambold** [comment]
When I pulled master and checked it actually did raise a duplicate error -- It was fixed in #336 🥇 

I wrote the tests anyway if you're interested in merging them #348 -- If not, nbd!

**Comment by kmax12** [comment]
fixed by #348. closing
